src/main/java/gov/nasa/pds/harvest/search/oodt/crawler/Crawler.java

```diff
@@ -0,0 +1,80 @@
+//// Copyright 2006-2017, by the California Institute of Technology.
```

Comment:
This looks like a _new_ file but is entirely commented-out … could this just not be part of the commit?